src/main/java/frc/robot/RobotContainer.java

```diff
@@ -91,40 +85,61 @@ public RobotContainer() {
    * {@link JoystickButton}.
    */
 
+  public static double getLeftXWithDeadband() {
+    return -MathUtil.applyDeadband(m_driverController.getLeftX(),
+      OIConstants.kDriveDeadband);
+  }
+
   public static double getDriverLeftX() {
-    return -MathUtil.applyDeadband(m_driverController.getLeftX(), OIConstants.kDriveDeadband);
+    return (.5)*Math.pow(
+      (getLeftXWithDeadband()), 5) + 
+      (.5)*(getLeftXWithDeadband());
```

Comment:
```suggestion
    return 0.5 * Math.pow(getLeftXWithDeadband(), 5) + 0.5 * getLeftXWithDeadband();
```
Did vscode format it like this? Try the format in the suggestion, looks cleaner imo.